File: lib/models/locations/supported_province_or_state.dart
// Collection Name /provinces_or_states/{provinceOrStateId}
// Branch : supported_locations_resources_crud ->  create_supported_locations_front_end
class SupportedProvinceOrState {
  String provinceOrStateNo;
  String provinceOrStateName;
  String countryFK;

  SupportedProvinceOrState({
    required this.provinceOrStateNo,
    required this.provinceOrStateName,
    required this.countryFK,
  });

  factory SupportedProvinceOrState.fromJson(dynamic json) =>
      SupportedProvinceOrState(
        provinceOrStateNo: json['provinceOrStateNo'],
        provinceOrStateName: json['provinceOrStateName'],
        countryFK: json['countryFK'],
      );

  @override
  String toString() {
    return provinceOrStateName;
  }
}

File: functions/models/locations/supported-province-or-state.js
/* eslint-disable*/
class SupportedProvinceOrState {

    constructor(countryFK, provinceOrStateName, provinceOrStateNo) {
        this.countryFK = countryFK;
        this.provinceOrStateName = provinceOrStateName;
        this.provinceOrStateNo = provinceOrStateNo;
    }

    toJson() {
        return {
            countryFK: this.countryFK,
            provinceOrStateName: this.provinceOrStateName,
            provinceOrStateNo: this.provinceOrStateNo,
        };
    }
}

export default SupportedProvinceOrState